cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -583,7 +583,16 @@
   public static final class Notification {
     public static final String TRANSPORT_SYSTEM = "notification.transport.system";
 
-    public static final String KAFKA_DELEGATE_NOTIFICATION_SERVICE = "kafka.delegate.notification.service";
+    /**
+     * Notifications in Streams constants.
+     */
+    public static final class Stream {
+      public static final String STREAM_HEARTBEAT_FEED_CATEGORY = "streamHeartbeats";
+      public static final String STREAM_FEED_CATEGORY = "streams";
+
+      /** Default number of bytes received by a stream after which a notification is sent */
+      public static final long DEFAULT_DATA_THRESHOLD = 1000000000;
```

Comment:
You can do something like `1 << 31` or `1024 * 1024 * 1024`, making it more obvious to be 1GB rather than counting zeros.